front-page.php

```diff
@@ -24,37 +24,50 @@
     if (!empty($works)) {
 ?>
 
-        <div id="home-work-holder" class="grid-row">
+        <div id="home-work-holder" class="grid-row padding-top-mid padding-bottom-large js-masonry margin-bottom-large">
 
 <?php
       foreach ($works as $work) {
         if (!empty($work['work_id'])) {
 
-          $work_image = get_the_post_thumbnail($work['work_id'], 'home-work', array('class'=>'home-work-image'));
+          $work_image = get_the_post_thumbnail($work['work_id'], 'full', false, array('data-no-lazysizes'=>true, 'class'=>'home-work-image'));
```

Comment:
`full` needs to be set to a registered size that makes sense for this design. I guess scale to width that is the max possible with this logic